### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by baer**
## Problem

Out of date dependencies were causing the following issues:

* Unable to `npm install` on Node 10
* [Deprecated `es2015` babel preset](https://babeljs.io/docs/en/env) is no longer getting fixes for compat
* Npm reports 421 vulnerabilities (386 low, 18 moderate, 15 high, 2 critical) in the project

## Fix

* Update dependencies
* Add `npm run lint` to the `package.json` - it was missing
* Add missing `jasmine` and `istanbul` dependencies to make `npm run coverage` work properly

### CODE REVIEW

File: .babelrc.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ["@babel/preset-env"]
```
Comment (baer) [inline .babelrc.js]: This change should be transparent. Babel's docs specify that:

> "By default, it has the same behavior as previous presets to compile ES2015+ to ES5"
https://babeljs.io/docs/en/env

The great thing about this plugin (if you haven't used it yet) is that you can add config to specify a support matrix and only the applicable presets will be applied. For example, if you only want to support down to IE9 and Node LTS you could easily do that.

File: package.json [hunk 2]
```diff
@@ -31,48 +31,50 @@
...
+    "gulp-istanbul": "^1.1.3",
     "gulp-jasmine": "^2.4.2",
     "gulp-load-plugins": "^1.5.0",
-    "gulp-sourcemaps": "^2.6.1",
-    "gulp-uglify": "^3.0.0",
+    "gulp-sourcemaps": "^2.6.4",
+    "gulp-uglify": "^3.0.1",
     "gulp-util": "^3.0.8",
+    "istanbul": "^0.4.5",
+    "jasmine": "^3.3.1",
```
Comment (baer) [inline package.json]: These two were included as transitive deps but without a top level include they are not available in the `node_modules/.bin` directory which means that the `npm run coverage` script wasn't working.

**Comment by BenjaminVanRyseghem** [comment]
Thank you :smile:

**Comment by BenjaminVanRyseghem** [comment]
in numbro 2.1.2

**Pull request merged**